app/main/css/global.css

```diff
@@ -15,6 +15,7 @@ body {
   height: 100vh;
   font-family: var(--main-font);
   overflow-y: hidden;
+  font-weight: 300;
```

Comment:
I'm not entirely fond of this addition, in my mind it makes the font look a bit too condensed.

![selection_294](https://cloud.githubusercontent.com/assets/1177900/22612070/912155de-ea6e-11e6-9ca1-8204e29879d9.png)